core/lexicon/en/setting.inc.php

```diff
@@ -50,7 +49,7 @@
 $_lang['setting_update'] = 'Update Setting';
 $_lang['settings_after_install'] = 'As this is a new install, you are required to control these settings, and change any that you may wish to. After you\'ve controlled the settings, press \'Save\' to update the settings database.<br /><br />';
 $_lang['settings_desc'] = 'Here you can set general preferences and configuration settings for the MODX manager interface, as well as how your MODX site runs. Double-click on the value column for the setting you\'d like to edit to dynamically edit via the grid, or right-click on a setting for more options. You can also click the "+" sign for a description of the setting.';
-$_lang['settings_furls'] = 'Friendly URLs';
+$_lang['settings_furls_gateway'] = 'Friendly URL & Gateway';
```

Comment:
```suggestion
$_lang['settings_furls_gateway'] = 'Friendly URLs & Gateway';
```
Plural variant more correct here.